src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/AuthenticationAgentActivity.cs

```diff
@@ -38,7 +38,9 @@ namespace Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.Platform
 {
     [Activity(Label = "Sign in")]
     [CLSCompliant(false)]
+#pragma warning disable CS3019 // CLS compliance checking will not be performed because it is not visible from outside this assembly
```

Comment:
>#pragma warning disable CS3019  [](start = 0, length = 31)

is this the right way to disable these? Is this just about a tool? It should be possible to configure outside the actual file

Reply:
This is the only method suggested by visual studio to disable this warning. I can look for more ways to do it.

Reply:
Is this the only place where we need this? If so that might be ok.

---
In reply to: [168056664](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/992#discussion_r168056664) [](ancestors = 168056664)

Reply:
I am able to consolidate suppressions that target warnings from analyzers, however, I cannot find a way to put general warning suppressions into the globalsuppressions file. Visual studio offers no option to do so. still researching how to do it.